lock-api/src/main/java/com/palantir/lock/LockClient.java

```diff
@@ -88,9 +87,7 @@ public String getClientId() {
     }
 
     @Override public String toString() {
-        return MoreObjects.toStringHelper(getClass().getSimpleName())
-                .addValue(isAnonymous() ? "anonymous" : ("\"" + clientId + "\""))
-                .toString();
+        return clientId;
```

Comment:
Is this break concerning?

Reply:
This break seems fine. No ones should have logic depending on the toString impl

Reply:
Yep. I'm not sure I like `LockClient.ANONYMOUS` being a completely empty string, but don't think we should block on that.